### PR TITLE
About snaps page

### PR DESCRIPTION
**Pull request by steverydz**
## Done
Updated layout of "About Snaps" page

## How to QA
- Go to https://snapcraft-io-4239.demos.haus/about
- Check it matches [the desktop design](https://app.zeplin.io/project/64107f86ec62b30f2acff2de/screen/64108869e051bb0dfde7496f)
- Check it matches [the mobile design](https://app.zeplin.io/project/64107f86ec62b30f2acff2de/screen/6421772d2b57ee2256a9557a)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-2958

### CODE REVIEW

**Comment by steverydz** [comment]
Hey @camillet16, could you please give this a design review?

File: templates/about/index.html [hunk 1]
```diff
@@ -7,115 +7,140 @@
...
-      With channels users can find their preferred balance between stability and the latest features. Though nothing is enforced, users expect the “Stable” channel to be well tested and updated less frequently, whereas the “Edge” channel is where they&rsquo;ll get the latest release as soon as it&rsquo;s available.
-    </p>
-    <a href="/docs/channels" class="p-button">Learn more about channels</a>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Official snaps from publishers</h2>
+    <div class="p-logo-section">
+      <div class="p-logo-section__items u-no-margin--bottom">
```
Comment (chillkang) [inline templates/about/index.html:59]: We might need those for images? What do you think?
<div class="row">
<div class="col-small-2 col-medium-3 col-2 col-logo u-align--center">

File: templates/about/index.html [hunk 2]
```diff
@@ -7,115 +7,140 @@
...
+        <div class="p-logo-section__item">
```
Comment (chillkang) [inline templates/about/index.html:60]: We might need those div tags for images? What do you think? 
`<div class="row">` `<div class="col-small-2 col-medium-3 col-2 col-logo u-align--center">`

Comment (steverydz) [reply templates/about/index.html:60]: I'd rather keep the Vanilla pattern unless there is good reason for breaking it

Comment (chillkang) [reply templates/about/index.html:60]: I see. For me, I see 4 images - 2 images in each row in mobile view so I assumed we would need them!

**Comment by camillet16** [comment]
Hi, Steve thanks for pinging me. Great work so far! here are some feedbacks :)

**Tabs:**
Screen Shot 2023-04-11 at 5 19 46 PM

- Remove margin-bottom on tabs.
- margin-bottom: 0rem;

----

**Hero: Body text**
Screen Shot 2023-04-11 at 5 20 28 PM

- remove each section's padding-top. The text already has an automatic padding-top.
- padding-bottom: 1rem on each paragraph

----

Screen Shot 2023-04-11 at 5 29 13 PM

- Please use these logos they're sized at 158x158 with an svg format with ample white space.
- Logos take up col-2 with 32px space between
kde-logo
mozilla-logo
jet-brains-logo
spotify-logo
google-logo
microsoft-logo

----

Screen Shot 2023-04-11 at 5 29 28 PM

- each strip has padding-bottom: 4rem

------

Screen Shot 2023-04-11 at 5 30 10 PM

- remove the padding-top from each strip which creates extra space between line and text.
- add padding-top: 0.625rem;  to image. Aligning this to text.


Future Thanks! 🎉

**Comment by steverydz** [comment]
@camillet16 Are you happy for this to be merged?

**Comment by camillet16** [comment]
@steverydz Sorry, the last thing is the space before the footer needs to be 128px in height. Other than that it's perfect 🌟 you got my +1 once that's good to go.

**Pull request merged**